### PR TITLE
Add a nightly system test against dev WFL

### PR DESCRIPTION
**Pull request by rexwangcc**
### Purpose

- No ticket is linked to this PR.

### Changes

- Add a nightly system test against dev WFL, which can also be triggered manually.

### Review Instructions

- Until we deploy the latest set of changes to `dev WFL`, this system will not pass :(

### CODE REVIEW

**Review by tbl3rd — APPROVED** [review]
Maybe add a `nightly` target and keep `system` for local testing?
I can help if that makes sense to you.

**Comment by rexwangcc** [comment]
> Maybe add a `nightly` target and keep `system` for local testing?
> I can help if that makes sense to you.

Done!

File: makerules/modules.mk [hunk 1]
```diff
@@ -14,23 +14,24 @@ DERIVED_MODULE_DIR := $(DERIVED_DIR)/$(MODULE)
...
 # Top level `make` targets for the module
-MAKE_TARGETS := prebuild format build unit integration check lint images system
+MAKE_TARGETS := prebuild format build unit integration check lint images system nightly
 
 # Timestamps for the top level make targets in a loose order of their timeings.
 # Implementers should write module make-targets against these, ensuring that
 # the time stamp is touched as the last action.
 # Example:
-# $(LINT): $(SCM_SRC) $(TEST_SCM_SRC)
+# $(LINT): $(SCM_SRC) $(TEST _SCM_SRC)
```
Comment (tbl3rd) [inline makerules/modules.mk:23]: oops

File: makerules/modules.mk [hunk 2]
```diff
@@ -14,23 +14,24 @@ DERIVED_MODULE_DIR := $(DERIVED_DIR)/$(MODULE)
...
 # 	$(PYTHON) -m pylint $(PYLINT_OPTIONS)
 # 	@$(TOUCH) $@
 
 PREBUILD    := $(DERIVED_MODULE_DIR)/prebuild.$(TIMESTAMP)
 LINT        := $(DERIVED_MODULE_DIR)/lint.$(TIMESTAMP)
-BUILD       := $(DERIVED_MODULE_DIR)/build.$(TIMESTAMP)   
-UNIT        := $(DERIVED_MODULE_DIR)/unit.$(TIMESTAMP)  
+BUILD       := $(DERIVED_MODULE_DIR)/build.$(TIMESTAMP)
+UNIT        := $(DERIVED_MODULE_DIR)/unit.$(TIMESTAMP)
```
Comment (tbl3rd) [inline makerules/modules.mk:30]: Thanks.

**Review by tbl3rd — APPROVED** [review]
I like it. See "oops".

File: makerules/modules.mk (hunk 1 above)
Comment (rexwangcc) [reply makerules/modules.mk:23]: oops

**Pull request merged**